README.md

```diff
@@ -92,6 +92,22 @@ Proceed the following steps to install Kubemarine manually on your environment:
 
 **Note:** Kubemarine debugging available via `kubemarine/__main__.py`.
 
+#### Kubemarine Docker building
```

Comment:
Let's move this topic into separate doc , like BuildProcess. I think in the README we just need to put a link into ghcr.io registry where any user can take a latest tested docker image. In case custom build is needed, this separate doc should be used.

Reply:
Link to ghcr.io was added , information about building image moved to additional doc file .